README.md

```diff
@@ -1,15 +1,68 @@
-# V Rising Dedicated Server Instructions
-The V Rising Dedicated Server is as it sounds a dedicated server application for the game [V Rising](https://store.steampowered.com/app/1604030/V_Rising/).
-
-# Download
-The V Rising Dedicated Server application can be downloaded from Steam, and is freely available to everyone. It can be found in the Tools section of your library. At the moment there is only a Windows version of the server available. Hopefully we can provide the Linux version soon.
 
+- [V Rising Dedicated Server](#v-rising-dedicated-server)
+- [Required System Specifications](#required-system-specifications)
+  - [Small Server Instance (~10 users)](#small-server-instance-10-users)
+- [Supported Platforms](#supported-platforms)
+- [Installation](#installation)
+  - [Installation through Steam](#installation-through-steam)
+  - [Installation through SteamCMD](#installation-through-steamcmd)
+  - [Installation and Update with SteamPS](#installation-and-update-with-steamps)
+    - [Process](#process)
+    - [Process to update the game using SteamPS](#process-to-update-the-game-using-steamps)
+- [Running the Server](#running-the-server)
+- [Configuring the Server](#configuring-the-server)
+- [Save Files](#save-files)
+  - [Backups](#backups)
+- [RCON](#rcon)
+
+# V Rising Dedicated Server
+The V Rising Dedicated Server is a dedicated server application for the game [V Rising](https://store.steampowered.com/app/1604030/V_Rising/).
+
+# Required System Specifications
+## Small Server Instance (~10 users)
+* 6GB of RAM
+* 2 CPU Cores 
+
+# Supported Platforms
+* Windows
+* Linux / Docker are not supported at this time
```

Comment:
Thats not true; it runs fine in  Docker

Reply:
Can you share Dockerfile please?

Reply:
> Thats not true; it runs fine in Docker

Running and supported are not the same thing.

Reply:
```suggestion
* Linux / Docker are not officially supported at this time
```